MAUI/DataForm/layout.md

```diff
@@ -0,0 +1,372 @@
+
+#### Using attributes
+
+{% tabs %}
+{% highlight C# %}
+private double? percentage;
+
+    [DataFormDisplayOptions(ShowLabel = false)]
+    public string Name { get; set; }
+
+{% endhighlight %}
+{% endtabs %}
+
+#### Using event
+
+{% tabs %}
+{% highlight C# %}
+
+ this.dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+
+        if (e.DataFormItem != null && e.DataFormItem.FieldName == "Name")
+        {
+            e.DataFormItem.PlaceholderText = "Enter your name";
+            e.DataFormItem.ShowLabel = false;
+        }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+## Label position
+
+Label can be positioned either top or left side of the editor. By using the [DataFormDefaultLayoutSettings.LabelPosition] property, you can layout the label associated with editor for data form.
+
+By default, the label will be positioned at left side of the editor.
+
+{% tabs %}
+{% highlight XAML %}
+<ContentPage 
+...
+             xmlns:dataForm="clr-namespace:Syncfusion.Maui.DataForm;assembly=Syncfusion.Maui.DataForm">
+    <dataForm:SfDataForm
+        x:Name="dataForm">
+        <dataForm:SfDataForm.DefaultLayoutSettings>
+            <dataForm:DataFormDefaultLayoutSettings
+                LabelPosition="Left"/>
+        </dataForm:SfDataForm.DefaultLayoutSettings>
+    </dataForm:SfDataForm>
+</ContentPage>
+
+{% endhighlight %}
+{% highlight C# %}
+
+this.dataForm.DefaultLayoutSettings.LabelPosition = DataFormLabelPosition.Top;
+
+{% endhighlight %}
+{% endtabs %}
+
+#### Changing particular editors label position
+
+The label position of particular editor can be changed by using the [DataFormDefaultLayoutSettings.LabelPosition] property in `DataFormItem`, and it will be handled in the `GenerateDataFormItem` event.
+
+{% tabs %}
+{% highlight XAML %}
+<ContentPage 
+...
+             xmlns:dataForm="clr-namespace:Syncfusion.Maui.DataForm;assembly=Syncfusion.Maui.DataForm">
+    <dataForm:SfDataForm
+        x:Name="dataForm"
+        DataObject="{Binding DataObjects}"
+        GenerateDataFormItem="OnGenerateDataFormItem">
+    </dataForm:SfDataForm>
+</ContentPage>
+
+{% endhighlight %}
+{% highlight C# %}
+
+this.dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+
+        if (e.DataFormItem != null && e.DataFormItem.FieldName == "Name")
+        {
+            e.DataFormItem.DefaultLayoutSettings = new DataFormDefaultLayoutSettings
+            {
+                LabelPosition = DataFormLabelPosition.Left,
+            };
+        }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+## Loading images for label
+
+Image can be loaded instead of label  by using [LeadingLabelIcon] property of `DataFormItem`.
```

Comment:
The image can be loaded instead of the label by using the [LeadingLabelIcon] property of the `DataFormItem.`

Reply:
Content changed